### ISSUE TITLE
Installer / Updater

### ISSUE THREAD

**Issue by SethClydesdale** [SethClydesdale/forumactif-edge#3]
An Installer and Updater will be essential to installing all the files and updating the forum when new versions are released. This will require AJAX to get files and check for updates via the Github repo. A JSON file might be best for compiling a list of the versions that have been released and installed.

The draft for the installer can be found [here](https://github.com/SethClydesdale/forumactif-edge/blob/master/javascripts/installer.js), the updater has not yet been developed. I'm considering to restrict updating and installation to founders, because they'll require high-level access to the admin panel.

**Comment by SethClydesdale** [comment]
With everything progressing nicely I've started getting to work on the installer and updater -- generally it'll be the control panel for the FAE. Since it needs to be embedable via HTML pages management, I've scrapped the old draft and readied an HTML file which will contain all the necessary elements.

The new installer can be found [here](https://github.com/SethClydesdale/forumactif-edge/blob/master/installer.html). It's currently incomplete, so I'd refrain from installing it on your forum unless you're looking to contribute to it. Once the responsiveness has been completed I'll make working on this full priority.

As the installer and updater is subject to change ( and will be lengthy ), it may be a good idea to have external files for the installation and uninstallation instructions. The update instructions have already been decided as such.
